bin/genomescope.R

```diff
@@ -0,0 +1,752 @@
+#!/usr/bin/env Rscript
+
+## GenomeScope: Fast Genome Analysis from Unassembled Short Reads
+##
+## This is the automated script for computing genome characteristics
+## from a kmer histogram file, k-mer size, and readlength
+
+## https://github.com/schatzlab/genomescope/
```

Comment:
the license should be specified in the header, see https://github.com/nf-core/crisprseq/blob/master/bin/cigar_parser.R#L7 for an example. The other information that you already have looks good 👍